eZ/Publish/Core/MVC/Symfony/SiteAccess/Matcher/URIElement.php

```diff
@@ -141,7 +141,7 @@ public function analyseURI($uri)
         if ($uri == $uriElements) {
             $uri = '';
         } elseif (strpos($uri, $uriElements) === 0) {
-            sscanf($uri, "$uriElements%s", $uri);
+            $uri = mb_substr($uri, mb_strlen($uriElements));
```

Comment:
Note: on Windows, `sscanf` does not handle properly multibyte strings. E.g. for `$uri = '/fre/utf8-with-accent/é/fre/à/à/fre/é'` it returns `'/utf8-with-accent/é/fre/�'` instead of `'/utf8-with-accent/é/fre/à/à/fre/é'`. AFAIK `sscanf` is not intended to work properly with multibyte strings, however couldn't find any relevant info on this.